pysmt/smtlib/parser.py

```diff
@@ -392,7 +400,7 @@ def _get_basic_type(self, type_name, params=None):
     def _get_var(self, name, type_name, params=None):
         """Returns the PySMT variable corresponding to a declaration"""
         typename = self._get_basic_type(type_name, params)
-        return self._current_env.formula_manager.Symbol(name=name,
+        return self.env.formula_manager.Symbol(name=name,
```

Comment:
Maybe we should use ```FormulaManager.get_symbol()```. The difference being that ```get_symbol()``` will return ```None``` if the symbol is not defined, while ```Symbol``` will declare it.